docs/SCHEMA.md

```diff
@@ -354,7 +385,8 @@ type BaseVariableFormat =
 type VariableFormat =
   | BaseVariableFormat
   | ListFormat<BaseVariableFormat>
-  | MapFormat<BaseVariableFormat, BaseVariableFormat>;
+  | MapFormat<BaseVariableFormat, BaseVariableFormat>
+  | FileFormat<'json'>;
```

Comment:
Added in the code and docs, defined as `export type BaseFileFormat = typeof VALID_FILE_FORMATS[number]` so that it'll always be in sync with our dynamic list of valid file formats.